Aztec/Classes/TextKit/TextView.swift

```diff
@@ -887,9 +887,13 @@ open class TextView: UITextView {
         let delay = 0.05
         DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
             let pristine = self.selectedRange
-            let updated = NSMakeRange(max(pristine.location - 1, 0), 0)
+            var location = max(pristine.location - 1, 0)
+            if pristine.location == location {
+                location = min(pristine.location + 1, self.storage.length)
```

Comment:
The method documentation is appreciated, however the reasoning behind the logic in this method is still not easy to understand:

1. Why are we moving location to `-1`?
2. What's the logical meaning of `pristine.location == location`?  Can we store that in a boolean with a more meaningful name?  (for example: `selectedRangeStartsAtPositionZero`, which is what I believe we're checking for)
3. Why are we moving the location forward if `pristine.location == location`?  This is really not clear to me.